pkg/cmd/node/node.go

```diff
@@ -0,0 +1,62 @@
+package node
+
+import (
+	"crypto/tls"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/dcos/dcos-core-cli/pkg/mesos"
+	"github.com/spf13/cobra"
+)
+
+// NewCommand creates the `core node` subcommand.
+func NewCommand(ctx api.Context) *cobra.Command {
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:   "node",
+		Short: "Display DC/OS node information",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, ok := ctx.EnvLookup(cli.EnvStrictDeprecations)
+			if !ok {
+				ctx.Deprecated("Getting the list of nodes from `dcos node` is deprecated. Please use `dcos node list`.")
+				return newCmdNodeList(ctx).RunE(cmd, args)
+			}
+			return cmd.Help()
+		},
+	}
+	cmd.Flags().BoolVar(&jsonOutput, "json", false, "Print in json format")
+
+	cmd.AddCommand(
+		newCmdNodeList(ctx),
+	)
+
+	return cmd
+}
+
+func mesosClient(ctx api.Context) (*mesos.Client, error) {
+	cluster, err := ctx.Cluster()
+	if err != nil {
+		return nil, err
+	}
+
+	baseURL, _ := cluster.Config().Get("core.dcos_url").(string)
+	if baseURL == "" {
+		baseURL = cluster.URL()
+	}
+
+	return mesos.NewClient(
+		httpclient.New(
+			baseURL,
+			httpclient.Logger(ctx.Logger()),
```

Comment:
Here the logger will never display HTTP logs, one thing we should do is have a helper function to create an httpclient for plugins, that will support the `DCOS_VERBOSITY` env var which gets passed to plugins on invocation.